code/modules/power/singularity/particle_accelerator/particle.dm

```diff
@@ -34,11 +34,12 @@
 	energy = 50
 
 /obj/effect/accelerated_particle/New(loc, dir = 2)
-	src.loc = loc
-	src.set_dir(dir)
-	spawn(0)
-		move(1)
+	set_dir(dir)
+	..()
 
+/obj/effect/accelerated_particle/Initialize()
+	. = ..()
+	move(1)

```

Comment:
This actually does need to be spawn()'d (or other solution) because  move() is a ...
oh.
`move()` is a `while(TRUE) ... sleep()` function... implemented as a *recursive* function.
ok.

Reply:
So does it or doesn't it?